src/frontend/main/translations.ts

```diff
@@ -0,0 +1,144 @@
+      legendInfoLine1: 'Color coded by amount',
+      legendInfoLine2: 'of symptoms reported',
+      legendInfoLine3: 'The size of the circle illustrates',
+      legendInfoLine4: 'population size',
+      mapInfo: `
+<p>The map shows symptoms the respondents have in various municipalities. Only municipalities with over 25 answers are included.</p>
+
+<p>You can view answers by municipality by clicking the circle or using search.</p>
+      `,
+      responses: 'Responses',
+      shareOfPopulation: 'Share of population',
+      totalResponses: 'Total responses',
+      lastUpdated: 'last updated',
+      other: 'Other',
+      noInformation: 'No information',
+      totalResponsesModalInfoText: 'Total responses: {{total}} ({{percentage}}% of the population)',
+      comparedToMunicipalityPopulation: 'Compared to municipality population',
+      ofPopulation: 'of the population',
+      notEnoughResponses: 'Not enough responses from this area yet',
+      municipality: 'Municipality',
+    },
+    navigation: {
+      map: 'Map',
+      survey: 'Survey',
+      info: 'Info',
+    },
+    aboutPage: {
+      content: `
+<0>Symptom Radar collects information about the coronavirus</0>
+
+<1>
+Oiretutka is a collaborative project between Helsingin Sanomat and digital engineering and innovation consultancy Futurice. It collects information about symptoms people have experienced all around Finland and disseminates this information to readers. Other Finnish medias are also taking part in collecting the information. The goal is to gain a better understanding of how the coronavirus might be spreading in Finland. The project follows open source principles. <2>The source code for the site is available here.</2>
+</1>
+
+<2>
+The information was collected using a questionnaire form. This site displays the results of the questionnaire.
+Tiedot ovat kerätty kyselylomakkeen avulla. Tällä sivustolla näytämme kyselyn tuloksia. <2>Read more about the privacy policy (in Finnish).</2>
+</2>
+      `,
+    },
+  },
+  fi: {
+    symptomLabels: {
+      // NOTE: the character "­" in "koronavirus­tartunnasta" is not a normal "-", but
+      // rather a "soft hyphen", added here via copy-pasting.
+      // "­" is used in conjunction with css style `hyphens: manual`
+      // Using &shy; (the html code for ­) seems to be too troublesome since it will be
+      // is escaped by default by both i18next and React.
+      corona_infection_suspicion: 'Epäilys koronavirus­tartunnasta',
+      corona_suspicion: 'Epäilys koronasta',
+      cough: 'Yskää',
+      fever: 'Kuumetta',
+      breathing_difficulties: 'Vaikeuksia hengittää',
+      muscle_pain: 'Lihaskipuja',
+      headache: 'Päänsärkyä',
+      sore_throat: 'Kurkkukipua',
+      rhinitis: 'Nuhaa',
+      stomach_issues: 'Vatsaoireita',
+      sensory_issues: 'Aistien heikkenemistä',
+      loss_smell_or_taste: 'Hajuaistin tai makuaistin heikkenemistä',
+    },
+    main: {
+      languageSelector: 'Language',
```

Comment:
I guess these are on purpose with the other language? If you need to change the language, doesn't really help to have a label in the language you don't understand 🤔 But what happens if we add another language 😅

Reply:
> But what happens if we add another language sweat_smile

I have not thought further than that :exploding_head: , but great point! : D 

Seems like using the current language would make sense